apps/web/src/app/settings/page.tsx

```
@@ -0,0 +1,81 @@
+  };
+
+  return (
+    <div className="max-w-lg">
+      <h1 className="text-2xl font-bold mb-6">プロフィール設定</h1>
+
+      <label className="block text-sm font-medium mb-1">
+        GitHub ユーザー名
+      </label>
+      <p className="mb-4 text-gray-500">{user.name}</p>
+
+      <label className="block text-sm font-medium mb-1">表示名</label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)